examples/extensions/horizontal-pod-autoscaling/README.md

```diff
@@ -105,50 +98,45 @@ Now that the server is running, we will create a horizontal pod autoscaler for i
 To create it, we will use the [hpa-php-apache.yaml](hpa-php-apache.yaml) file, which looks like this:
 
 ```yaml
-apiVersion: extensions/v1beta1 
+apiVersion: extensions/v1beta1
 kind: HorizontalPodAutoscaler
 metadata:
   name: php-apache
   namespace: default
 spec:
-  maxReplicas: 10
-  minReplicas: 1
   scaleRef:
     kind: ReplicationController
     name: php-apache
     namespace: default
-  target:
-    quantity: 100m
-    resource: cpu
+  minReplicas: 1
+  maxReplicas: 10
+  cpuUtilization:
+    targetPercentage: 50
 ```
 
 This defines a horizontal pod autoscaler that maintains between 1 and 10 replicas of the Pods
 controlled by the php-apache replication controller you created in the first step of these instructions.
 Roughly speaking, the horizontal autoscaler will increase and decrease the number of replicas
-(via the replication controller) so as to maintain an average CPU utilization across all Pods of 100 millicores.
+(via the replication controller) so as to maintain an average CPU utilization across all Pods of 50%
+(please note, that in [rc-php-apache.yaml](rc-php-apache.yaml) each pod requests 200 milli-cores).
```

Comment:
the meaning of this comment is non-obvious
instead I would say:
(since each pod requests 200 milli-cores [rc-php-apache.yaml](rc-php-apache.yaml), this means average CPU utilization of 100 milli-cores)